tailwind.config.js

```diff
@@ -33,6 +33,7 @@ const spacing = {
   248: "24.8rem",
   356: "35.6rem",
```

Comment:
Maybe you could remove values that aren't used? There's no reason to keep redundant code around.

Reply:
Done